habitat_sim/robots/fetch_robot.py

```diff
@@ -0,0 +1,86 @@
+import magnum as mn
+import numpy as np
+
+from habitat_sim.physics import JointMotorSettings
+from habitat_sim.robots.mobile_manipulator import (
+    MobileManipulator,
+    MobileManipulatorParams,
+)
+
+
+class FetchRobot(MobileManipulator):
+    def __init__(self, urdf_path, sim, limit_robo_joints=True):
+        fetch_params = MobileManipulatorParams(
+            arm_joints=list(range(15, 22)),
+            wheel_joints=[2, 4],
+            arm_init_params=[
+                -0.45,
+                -1.08,
+                0.1,
+                0.935,
+                -0.001,
+                1.573,
+                0.005,
+                0.00,
+                0.00,
+            ],
+            ee_offset=mn.Vector3(0.08, 0, 0),
+            ee_link=22,
+            ee_constraint=np.array([[0.4, 1.2], [-0.7, 0.7], [0.25, 1.5]]),
+            arm_cam_offset_pos=mn.Vector3(0, 0.0, 0.1),
+            head_cam_offset_pos=mn.Vector3(0.17, 0.0, 1.2),
+            head_cam_look_pos=mn.Vector3(1, 0.0, 0.75),
+            gripper_joints=[23, 24],
+            gripper_closed_state=0.04,
+            gripper_open_state=0.0,
+            arm_mtr_pos_gain=0.3,
+            arm_mtr_vel_gain=0.3,
+            arm_mtr_max_impulse=10.0,
+            wheel_mtr_pos_gain=0.0,
+            wheel_mtr_vel_gain=1.3,
+            wheel_mtr_max_impulse=10.0,
+            base_offset=mn.Vector3(0.0, 0.0, 0.0),
+            ctrl_freq=30,
+        )
+        super().__init__(fetch_params, urdf_path, sim, limit_robo_joints)
+        self.back_joint_id = 6
+        self.head_rot_jid = 8
+        self.head_tilt_jid = 9
+
+    def reconfigure(self) -> None:
+        super().reconfigure()
+        # remove any default damping motors
+        for motor_id in self._robot.existing_joint_motor_ids:
```

Comment:
If this all isn't specific for the Fetch robot only we can move it to Mobile Manipulator `reconfigure`.

Reply:
Depends on whether we want to use the default damping motors from URDF or clear them for custom settings. This seemed like a top level decision so we could call super methods. 🤔 
Let's see how we feel once we try a couple more robots.